.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 pkg/*.gem
+Gemfile.lock
```

Comment:
done, I'm just wondering why do you personally prefer to keep `Gemfile.lock` in git repository? :)